spec/features/board_pages/board_pages_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
```

Comment:
This file seems quite different form Pro.
And maybe it should be more similar or ideally the same file.

Example: this code also works in CE: `create(:board, node: current_project.methodology_library, project: current_project)`

Reply:
how is this being tracked so it doesn't slip if the PR is merged?

Reply:
> how is this being tracked so it doesn't slip if the PR is merged?

Notes/tasks to make sure in the next scope where they become appropriate they get re-added. It's an imperfect solution.

In this particular case the next scope already existed and we made sure the port existed in the next PR.